package-lock.json

```diff
@@ -2092,8 +2092,8 @@
       "integrity": "sha512-euqiYT4HCvL6lk3BgQt7Psz2im8E+fzQVxBaPbRsLDp71sCKNZ2cPNxk/VM1xIKunhjS990lKU0jCF/hfvqjpA==",
       "dev": true,
       "requires": {
-        "emptykit.css": "1.0.1",
-        "styled-components": "3.4.5"
+        "emptykit.css": "^1.0.1",
```

Comment:
accidentally checked in?

Reply:
Must have been updated when doing the npm install for `brave-ui`; I think it's safe to stay. Thanks for checking though 😄